server/server.go

```diff
@@ -590,6 +592,28 @@ func (s *Server) sendError(reply string, err error) {
 	s.publish(reply, proxy.NewErrorResponse(err))
 }
 
+func (s *Server) logEventWithTimestamp(e ari.Event, ts *int64) {
+	eventTs := time.Now().Unix()
+	if *ts == 0 {
+		s.Log.Debug("event received", "kind", e.GetType(), "time received", eventTs)
+	} else {
+		s.Log.Debug("event received", "kind", e.GetType(), "time received", eventTs,
+			"delta of this event's timestamp and the previous event's timestamp", eventTs-*ts)
```

Comment:
I think "received" and "delta" are sufficient and makes it easier to process in structured log processing.